docs/CaliptraIntegrationSpecification.md

```diff
@@ -194,16 +194,18 @@ Although fuse values (and the fuse done register) persist across a warm reset, S
 
 ## Interface rules
 
-The following figure shows the reset rules and timing.
+The following figure shows the reset rules and timing for cold boot flows.
 
 *Figure 3: Reset rules and timing diagram*
 
 ![](./images/Caliptra_reset_timing.png)
 
-Deassertion of cptra\_pwrgood indicates a power cycle that results in returning Caliptra to its default state. All resettable flops are reset.
+De-assertion of cptra\_pwrgood indicates a power cycle that results in returning Caliptra to its default state. All resettable flops are reset.
```

Comment:
nit: recommend no hyphen because most often used in this doc is unhyphenated form.